commit 19533920a6ebef1b4cc581c4a76ff15602940ec2
Author: John Levon <john.levon@joyent.com>
Date:   2019-07-08T09:52:33+00:00 (3 months ago)
    
    joyent/node-zkstream#9 use var instead of const so we can run in strict mode

diff --git a/README.md b/README.md
index ca63117..30c4a3d 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,7 @@
 zkstream
 ========
 
+
 A minimal streams-based node client for the ZooKeeper protocol.
 
 API
